dotnet-desktop-guide/framework/winforms/controls/button-control-windows-forms.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Button Control"
+description: Learn about the button control in Windows forms, which allows the user te click a button in order to perform an action.
```

Comment:
Typo, should be "to click" instead of "te click"

Reply:
Addressed.